site/cypress/integration/autograding_status_1.spec.js

```diff
@@ -91,7 +94,15 @@ skipOn(Cypress.env('run_area') === 'CI', () => {
             cy.get('#autograding-status-table tbody tr td').eq(9).then(element => cy.get(element).should('contain', '0'));
             cy.get('#autograding-status-table tbody tr td').eq(10).then(element => cy.get(element).should('contain', '102'));
 
-            cy.logout();
+        });
+
+        it('should only allow instructor level users', () => {
+            // attempt to visit page as student
+            cy.visit([]);
+            cy.login('student');
+            cy.visit(autograding_status_path);
+            cy.get('#autograding-status-table').should('not.exist');
+            cy.visit('/');
         });
```

Comment:
Should checks be added for graders and TAs?  Are there any differences between regular instructors and faculty which should be tested?

Reply:
Right now the page is available for users that are made an instructor for any course. However, there is no users that are instructors and not faculty in our set up right now (?).

Reply:
Ok, that seems like something to add to the sample course in the future but not necessarily in this PR then.  Perhaps just check the access permissions for graders and TAs and then I think it should be good to go.